### ISSUE TITLE
Question about key body tracking(local or global)

### ISSUE THREAD

**Issue by love1994love** [jimazeyu/exbody2#10]
In the paper "ExBody2: Advanced Expressive Humanoid Whole-Body Control”, it says that it uses local keypint tracking instead of global key point tracing. However, in the released code, the tracking key body function is defined as:
'''

    def _reward_tracking_demo_key_body(self):
        demo_key_body_pos_local = self._curr_demo_keybody.view(self.num_envs, self._num_key_bodies, 3)
        if self.cfg.motion.global_keybody:
            curr_demo_xyz = torch.cat((self.target_pos_abs, self._curr_demo_root_pos[:, 2:3]), dim=-1)
        else:
            curr_demo_xyz = torch.cat((self.root_states[:, :2], self._curr_demo_root_pos[:, 2:3]), dim=-1)
        demo_global_body_pos = local_to_global(self._curr_demo_quat, demo_key_body_pos_local, curr_demo_xyz).view(self.num_envs, -1)
        cur_global_body_pos = self.rigid_body_states[:, self._key_body_ids_sim[self._key_body_ids_sim_subset], :3].view(self.num_envs, -1)
        rew = torch.exp(-torch.norm(cur_global_body_pos - demo_global_body_pos, dim=1))
        return rew

'''

Why not use local_body_pose in this reward?
